library/src/main/kotlin/com/a99/rxplaces/Extensions.kt

```diff
@@ -0,0 +1,14 @@
+package com.a99.rxplaces
+
+import android.location.Location
+
+internal fun <T> List<T>.format(): String? {
```

Comment:
Please provide a more described name for this one, such as `insertPipeBetweenValues()` or something like that